src/app/cube/content-pages/contribute-page/contribute-page.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { sections } from './copy';
+
+@Component({
+  selector: 'clark-contribute-page',
+  templateUrl: './contribute-page.component.html',
+  styleUrls: ['./contribute-page.component.scss']
+})
+
+export class ContributePageComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void { }
+  }
```

Comment:
Somethings incorrect in this file. This '}' should not be hear and the code indentation is not correct on the next few lines